basic_samples/SDS/Java/readme.rst

```diff
@@ -366,8 +366,8 @@ The following shows how this is done in the code:
 The process consists of two steps. First, the Property Override must be created, then the
 stream must be updated. Note that the sample retrieves three data points
 before and after updating the stream to show that it has changed. See
-the `Sds documentation <https://cloud.osisoft.com/documentation>`__ for
-more information about Sds Property Overrides.
+the `SDS documentation <https://cloud.osisoft.com/documentation>`__ for
```

Comment:
link not working https://ocs-docs.osisoft.com/Documentation/SequentialDataStore/Data_Store_and_SDS.html